.github/workflows/check-dependencies.yml

```diff
@@ -0,0 +1,32 @@
+name: Check Dependencies
+
+on:
+  schedule:
+    - cron: '00 0 * * *'
+
+jobs:
+  check-language-servers:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        package:
+          - salesforcedx-vscode-lwc
+          - salesforcedx-vscode-lightning
+    steps:
+      - uses: actions/checkout@master
+      - name: set remote url
+        run: git remote set-url --push origin https://$GITHUB_ACTOR:${{ secrets.GITHUB_TOKEN }}@github.com/$GITHUB_REPOSITORY
+      - name: package-update
+        uses: taichi/actions-package-update@master
+        env:
+          AUTHOR_EMAIL: actions@github.com
+          AUTHOR_NAME: github-actions
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
The current github token provides more access than the one required by this action (creating pull requests). We should change this to use a token that only grants pull request creation access.

Reply:
Would you be able to add a less powerful token to the repo secrets and provide me with the name?